docs/jwst/refpix/description.rst

```diff
@@ -136,6 +139,9 @@ the reference pixels in each group and subtracted from each pixel.
 If the science dataset has at least 1 group with no valid reference pixels,
 the step is skipped and the S_REFPIX header keyword is set to 'SKIPPED'.
 
+The ``use_side_ref_pixels``, ``side_smoothing_length``, ``side_gain`` and
```

Comment:
Since you're usually the backtics to style the argument names here, we should probably use them in other places too (like line 126 above), for consistency.